community/cypher/cypher-compiler-3.0/src/test/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/NodeHashJoinPlanningIntegrationTest.scala

```diff
@@ -60,34 +60,6 @@ class NodeHashJoinPlanningIntegrationTest extends CypherFunSuite with LogicalPla
     result should equal(expected)
   }
 
-  test("Should build plans with leaves for both sides if that is requested by using hints") {
```

Comment:
Now that I think of it, this test is weird. The greedy planner only supported one index hint. How does this even run...?

Currently, supplying two index hints for a single pattern relationship will fallback to rule planner, which I guess is fine.